prov/util/src/util_mem_monitor.c

```diff
@@ -606,6 +612,80 @@ static void *ofi_uffd_handler(void *arg)
 				(void *) (uintptr_t) msg.arg.remap.from,
 				(size_t) msg.arg.remap.len);
 			break;
+		case UFFD_EVENT_PAGEFAULT:
+
+			 /* The event tells us the address of the fault
+			  * (which can be anywhere on the page). It does not
+			  * tell us the size of the page so we have to guess
+			  * from the list of known page_sizes.
+			  *
+			  * We employ the standard resolution: install a zeroed page.
+			  */
+
+			address = (void *) (uintptr_t) msg.arg.pagefault.address;
+			found = false;
+
+			for (i = 0; i < num_page_sizes; ) {
+				/* setup a zeropage reqest for this pagesize */
+				zp.range.start = (uint64_t) (uintptr_t)
+					ofi_get_page_start(address, page_sizes[i]);
+				zp.range.len = (uint64_t) page_sizes[i];
+				zp.mode = 0;
+				zp.zeropage = 0;
+
+				ret = ioctl(uffd.fd, UFFDIO_ZEROPAGE, &zp);
+
+				if (0 == ret) {		/* success */
```

Comment:
Please put the constant at the right side of the comparison.  Not about which way is better, but to keep the convention consistent. Same for other similar instances.

Reply:
Will do. Also, I think this PR can be squashed into a single commit.

Reply:
Please also fix those similar places that this comment applies to.

Reply:
Done.